server/middlewares/BookValidator.js

```diff
@@ -164,4 +214,34 @@ export default class BookValidator {
       checkForErrors,
     ];
   }
+
+  /**
+ * Edit validator
+ * @returns {array} an array of check API middleware
+ * @memberof Validation
+ */
+  static EditValidation() {
+    return [
+      BookValidator.checkNumber('id'),
+      BookValidator.optionalTitle(),
+      BookValidator.optionalDescription(),
+      BookValidator.optionalIsbn(),
+      BookValidator.optionalPrice(),
+      BookValidator.optionalYearPublished(),
+      BookValidator.optionalAuthorName(),
+      checkForErrors,
+    ];
+  }
+
+  /**
+ * Delete validator
+ * @returns {array} an array of check API middleware
+ * @memberof Validation
+ */
+  static DeleteValidation() {
```

Comment:
The name DeleteValidation seems  **ambiguous** too

Reply:
is already under the book validation class, there is no need to include book to it, the name is already long why make it longer